proto/validator/accounts/v2/web_api.proto

```diff
@@ -371,3 +386,12 @@ message DeleteAccountsResponse {
     // List of public keys successfully deleted.
     repeated bytes deleted_keys = 1;
 }
+message ExportSlashingProtectionResponse{
+    // JSON representation of the slash protection
+    string file = 1;
+}
+
+message ImportSlashingProtectionRequest{
+    // JSON representation of the slash protection
+    string slashingProtectionJSON = 1;
```

Comment:
The format for protobuf file naming of fields should be snake_case instead of camelCase, so we can call it slashing_protection_json and same for the field in the export response

Reply:
Missed that.  Done now